PacketDotNet/VxlanPacket.cs

```diff
@@ -0,0 +1,131 @@
+/*
+This file is part of PacketDotNet.
+
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using PacketDotNet.Utils;
+using PacketDotNet.Utils.Converters;
+
+namespace PacketDotNet
+{
+    /// <summary>
+    /// Vxlan
+    /// </summary>
+    public sealed class VxlanPacket : Packet
+    {
+        /// <summary>
```

Comment:
Just commenting on this file in general, can you reformat? There's some additional whitespace (two white lines) and also the XML comments are not according to code style, e.g. the values should be on one line where possible, see here: https://github.com/dotpcap/packetnet/blob/af442b608f72d0c3f19ed4a9dc32f950558fde48/PacketDotNet/DhcpV4Packet.cs#L26

Reply:
Is coding style with or without new line at EOF? You added new line to GreNvgrePacketTest.cs but DhcpV4Packet.cs doesn't have it.

Reply:
I edited it through GitHub, I think that added the character. It doesn't really matter as long as the rest of the class is OK, it seems now 👍